cl/assets/tailwind/input.css

```
@@ -89,7 +89,7 @@
   }
 
   .alert-warning {
-    @apply alert text-yellow-700 bg-yellow-100 border-yellow-500;
+    @apply alert text-yellow-700 bg-yellow-50 border-amber-450 flex border-2 rounded-2xl gap-[10px] items-center justify-center p-4 self-stretch;
```

I know there's a bunch of `-[10px]` scattered around the new templates 🫠 but let's try to use actual spacing values going forward.

`10px` is `2.5` in [tailwind's default spacing scale](https://v3.tailwindcss.com/docs/customizing-spacing#default-spacing-scale) so we can do:

```suggestion
    @apply alert text-yellow-700 bg-yellow-50 border-amber-450 flex border-2 rounded-2xl gap-2.5 items-center justify-center p-4 self-stretch;
```